Makefile

```diff
@@ -40,13 +42,13 @@ clean: stop reset-permissions
 setup: stop setup-containers setup-certificates setup-dependencies reset-permissions
 setup-containers:
 	$(info Building containers)
-	@docker-compose build deps js css web db api
+	@docker-compose build
```

Comment:
Was able to remove this explicit (here and below) list now that these are the only services in `docker-compose.yml`.